benchmark/super_resolution_model_zoo/README.md

```diff
@@ -0,0 +1,38 @@
+# Benchmarks
```

Comment:
could we generalize this sub project to other models other than SR, which would be very useful feature?

Reply:
Yes i put a TODO in the main index.js file to improve this benchmark to take arbitrary models and configs. For that, we would need a benchmark driver similar to our test-runner-cli. I will do a followup change just for the benchmarking tool.